guidelines/monthly.md

```diff
@@ -1,27 +1,47 @@
 # BarcelonaJS Monthly events guidelines.
 
-About
+## About
 
 - BarcelonaJS will celebrate one event per month
-- The event is hosted by an external entity/company
+- The event is hosted by an external entity/company referred as Sponsor
 - The event should start around 19:00
 - The event must be free
-- BarcelonaJS won't charge anything for organizing the event.
-- All participants and organizers should agree with our CoC
-- The external entity can do an intro talk limited to 5m
+- BarcelonaJS won't charge anything for organizing the event
+- All attendees, BarcelonaJS and Sponsor should agree with our [CoC](../code-of-conduct.md)
+- The Sponsor can do an intro talk limited to 5m
+- The Sponsor should provide with a venue of at least 100 seats.
 
-Talks:
+## Talks requirements:
 
 - Talks must be in English
-- Talks should be in the 20-60m range.
-- Talks should be frontend related.
-
-Required information:
-
-- Company must provide the information 3 weeks before the event.
-- Company must provide company name
-- Company must provide venue address.
-  - Ideally google maps link.
-- Company must indicate venue capacity.
-- Company should sponsor food/drinks.
-- Company should provide talk brief and speaker twitter/linkedin
+- Talks should be in the 20-30m range.
+- Talks should be JS or Frontend related.
+
+## Event agenda:
+
+- Event introduction 5m (BarcelonaJS)
+- Sponsor introduction 5m (Sponsor)
+- First talk 20m - 30m
+- Break 5m (Non recruiter attendees can pitch that their company is hiring)
+- Second talk 20m - 30m
+- Networking with drinks and food 20m - 1h
+
+## Sponsor requirements:
+
+- Sponsor must provide the information 3 weeks before the event.
+- Sponsor must provide company name
+- Sponsor must provide venue address. **Ideally, Google Maps link**
+- Sponsor must indicate venue capacity.
+- Sponsor should sponsor food/drinks.
+- Sponsor should provide talk brief and speaker Twitter/LinkedIn
+
+## Social strategy
+
+- Post Meetup event (🚧 create template and link to it) from BarcelonaJS account
+- Tweet meetup event (🚧 create template and link to it) from BarcelonaJS account
```

Comment:
```suggestion
- Tweet meetup event (🚧 create template and link to it) from BarcelonaJS account
- Tweet promoting the first speaker (🚧 create template and link to it) from BarcelonaJS account
- Tweet promoting the second speaker (🚧 create template and link to it) from BarcelonaJS account
```

I would suggest doing it in 3 tweets:
 - Meetups announcement
 - Speaker 1 announcement
 - Speaker 2 announcement
This way we would help to promote speakers a bit more